StereoKit/Math/Matrix.cs

```diff
@@ -444,6 +444,32 @@ public static Matrix Perspective(Vec2 imageResolution, Vec2 focalLength, float n
 			return Matrix4x4.CreatePerspective(nearPlaneDimensions.x, nearPlaneDimensions.y, nearClip, farClip);
 		}
 		
+		/// <summary>This creates a matrix used for projecting 3D geometry
+		/// onto a 2D surface for rasterization. With the known camera 
+		/// intrinsics, you can replicate its perspective!</summary>
+		/// <param name="imageResolution">The resolution of the image. This
+		/// should be the image's width and height in pixels.</param>
+		/// <param name="focalLength">The focal length of the camera in pixels,
+		/// with image coordinates +X (pointing right) and +Y (pointing up).</param>
+		/// <param name="principalPoint">The principal point of the camera in pixels,
```

Comment:
What is a principal point? I'm a little fuzzy on the idea.

Reply:
It's usually in the center of the image, but for some cases, there's an offset from where the line (i.e., optical axis) going straight out from the camera hits the image plane